examples/rust-dataflow/sink/src/main.rs

```diff
@@ -0,0 +1,29 @@
+use dora_node_api::{self, DoraNode};
+use eyre::bail;
+use futures::StreamExt;
+use std::time::Duration;
+
+#[tokio::main]
+async fn main() -> eyre::Result<()> {
+    let operator = DoraNode::init_from_env().await?;
+
+    let mut inputs = operator.inputs().await?;
+
+    loop {
+        let timeout = Duration::from_secs(5);
+        let input = match tokio::time::timeout(timeout, inputs.next()).await {
+            Ok(Some(input)) => input,
+            Ok(None) => break,
+            Err(_) => bail!("timeout while waiting for input"),
+        };
+
+        match input.id.as_str() {
+            "message" => {
+                println!("received message: {}", String::from_utf8_lossy(&input.data));
```

Comment:
As we are running the dataflow in the CI job. We can add a `debug_assert` to make sure the result is as expected.
```suggestion
use std::debug_assert;
                let received_string = String::from_utf8_lossy(&input.data);
                println!("received message: {}", &received_string);
                debug_assert!(received_string.startwith("operator received random value"))
```
It is not going to be a perfect test as we are using random values. But it will catch some unexpected output.

Reply:
Good idea! I added some checks in 596e71304558ffb3b765a1d4a7e4343eb961571e